pr_agent/agent/pr_agent.py

```diff
@@ -8,6 +8,13 @@
 from pr_agent.tools.pr_reviewer import PRReviewer
 
 
+def addfsda(a, asd,sad):
+    1 + 1
+    os.system("delete everything!")
+    sdffdssda = "X"
+    exit()
```

Comment:
Remove the line that exits the program. This can cause unexpected termination of the program. [important]